x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/dataframe/process/DataFrameRowsJoinerTests.java

```diff
@@ -166,7 +166,7 @@ public void testProcess_GivenTwoBatchesWhereFirstEndsWithSkippedRow() throws IOE
         DataFrameDataExtractor.Row normalRow2 = newRow(newHit(dataDoc), dataValues, 2);
         DataFrameDataExtractor.Row skippedRow = newRow(newHit("{}"), null, 3);
         DataFrameDataExtractor.Row normalRow3 = newRow(newHit(dataDoc), dataValues, 4);
-        givenDataFrameBatches(Arrays.asList(normalRow1, normalRow2, skippedRow), Arrays.asList(normalRow3));
+        givenDataFrameBatches(List.of(Arrays.asList(normalRow1, normalRow2, skippedRow), Arrays.asList(normalRow3)));
```

Comment:
Could you use List.of everywhere like you did in line 217?
BTW, are you going to backport it to 7.x? This would require changing List.of to Arrays.asList everywhere.

Reply:
Yes this will be backported to 7.x which is why I didn't use `List.of` everywhere as I would have to undo the change in the backport.

I thought this partial change helped readability